tests/functional/test_main.py

```diff
@@ -1,7 +1,10 @@
-import commands
+from future import standard_library
```

Comment:
Gotcha. Thanks. Added it. I should probably get in the habit of putting packages in the requirements files as soon as I import them somewhere.